.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+Linked PRs: (links to corresponding PRs, optional)
+
+## Changes
+- (changes here)
+
+## TODOs:
```

Comment:
Let's move TODOs to the bottom - the TODO list is mainly for the for the person opening the PR, not the reviewer. We should be kinder to the reviewer by putting all of the important information they need up top (what changed, how to test).